frontend/src/components/gradebook/gradebook.tsx

```diff
@@ -23,6 +32,24 @@ const Gradebook = () => {
     }
   };
 
+  const searchStudent = (event: any) => {
+    setStudents(data.students.filter((student: StudentInfo) => student.name.toLowerCase().includes(event.target.value.toLowerCase())));
```

Comment:
idk why we don't have `prettier` but let's make this multiline

Reply:
I can add a .prettierrc